client/spec/authoring_spec.js

```diff
@@ -102,4 +102,30 @@ describe('authoring', function() {
         });
 
     });
+
+    describe('subnavigation menu', function () {
+        it('allows to create a new empty package', function () {
+            // 1. Navigate to the authoring main page.
+            //
+            // 2. Open the "plus icon" menu in subnav bar and click the
+            //    "Empty Package" link in the menu.
+            //
+            // 3. Get the newly created item's info box and check if it
+            //    contains an icon indicating that the item's type is indeed
+            //    "composite".
+            //
+            authoring.openAuthoringHome();
+
+            authoring.navbarMenuBtn.click();
+
+            authoring.newEmptyPackageLink.click().then(function () {
+                return authoring.infoIconsBox.all(
+                    by.css('.filetype-icon-composite')
```

Comment:
Yeah, that's on purpose. The package icon is not some common, always present page element, it's specific to this test case. It's a dynamic thing that might or might not exist as a result of previous actions, and since it also represents the meat of the test, it is referenced explicitly.

FWIW, I saw the same pattern in the existing tests...